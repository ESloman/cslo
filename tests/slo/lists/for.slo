var list = [1, 2, 3, 4, 5];
for (var x = 0; x < len(list); x++) {
    println(list[x]);
}
println(list);
