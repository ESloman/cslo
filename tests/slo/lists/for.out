1
2
3
4
5
list[5]: [1, 2, 3, 4, 5]
